Dockerfile

```
@@ -1,7 +1,11 @@
-FROM centos:7.8.2003
+FROM --platform=linux/amd64 golang:1.15 AS entrykit
+RUN go get -v -ldflags "-s -w" github.com/progrium/entrykit/cmd
+
+FROM --platform=linux/amd64 centos:7.8.2003
```

せっかくなのでcentosを最新にしたい
https://hub.docker.com/_/centos?tab=description
(centos8はなんかすぐサポート切れるらしいんでcentos7のままでいいかな・・・）
```suggestion
FROM --platform=linux/amd64 centos:7.9.2009
```

---

centos8が7よりもサポート早く切れるんですね・・・
suggestionしてもらった内容でコミットします！